src/lib/binary.ts

```diff
@@ -6,6 +6,8 @@ import {
 } from "./error";
 import { getFetcher } from "./getFetcher";
 
+const OSX_ONLY_ROUTER_VERSIONS = ["v1.38.0", "v1.39.0"];
```

Comment:
This isn't the right name for this variable.

```suggestion
// These versions of the Router did not have x86 pre-built versions of the Router available for distribution.
const OSX_ARM_ONLY_ROUTER_VERSIONS = ["v1.38.0", "v1.39.0"];
```